libs/documentation/src/lib/storybook/formly/formly-text-area/formly-text-area.stories.ts

```diff
@@ -0,0 +1,205 @@
+import { FormlyTextAreaConfigurableModule } from './formly-text-area-configurable/formly-text-area-configurable.module';
+import { FormlyTextAreaDescriptionModule } from './formly-text-area-description/formly-text-area-description.module';
+import { FormlyTextAreaLabelModule } from './formly-text-area-label/formly-text-area-label.module';
+import { FormlyTextAreaPlaceholderModule } from './formly-text-area-placeholder/formly-text-area-placeholder.module';
+import { FormlyTextAreaRequiredModule } from './formly-text-area-required/formly-text-area-required.module';
+import { FormlyTextAreaMaxlengthModule } from './formly-text-area-maxlength/formly-text-area-maxlength.module';
+
+const disable = {
+  table: {
+    disable: true,
+  },
+};
+const templateOptions = {
+  table: {
+    category: 'template-options',
+  },
+};
+
+export default {
+  title: 'Formly/Text Area',
+  decorators: [
+    moduleMetadata({
+      imports: [
+        CommonModule,
+        ReactiveFormsModule,
+        SdsFormlyModule,
+        FormsModule,
+        FormlyModule.forRoot(),
+        FormlyTextAreaIntroductionModule,
+        FormlyTextAreaConfigurableModule,
+        NoopAnimationsModule,
+        FormlyTextAreaLabelModule,
+        FormlyTextAreaDescriptionModule,
+        FormlyTextAreaMaxlengthModule,
+        FormlyTextAreaRequiredModule,
+        FormlyTextAreaPlaceholderModule,
+      ],
+    }),
+  ],
+  argTypes: {
+    label: templateOptions,
+    placeholder: templateOptions,
+    description: templateOptions,
+    required: templateOptions,
+    maxLength: templateOptions,
+    //disabled: templateOptions,
+    //tooltipText: templateOptions,
+  },
+} as Meta;
+
+const formlyConfigFunction = (
+  label: string,
+  placeholder: string,
+  description: string,
+  required: boolean,
+  maxLength: number
+  //disabled: boolean,
+  //tooltipText: string
+) => {
+  return {
+    label: label ?? '',
+    placeholder: placeholder ?? '',
+    description: description ?? '',
+    required: required ?? false,
+    maxLength: maxLength ?? '',
+  };
+};
+const Template: Story = (args) => {
+  const { label, placeholder, description, required, maxLength } = args;
+  let config = formlyConfigFunction(label, placeholder, description, required, maxLength);
+  return {
+    template: '<sds-formly-text-area-configurable [config]="configObj"></sds-formly-text-area-configurable>',
+    props: {
+      configObj: config,
+      ...args,
+    },
+  };
+};
+
+export const Introduction: Story = (args) => ({
+  template: '<sds-formly-text-area-introduction></sds-formly-text-area-introduction>',
+  props: args,
+});
+Introduction.parameters = { options: { showPanel: false } };
+
+export const Configurable = Template.bind({});
+Configurable.args = {
+  label: 'Entity Description',
+  placeholder: 'eg: Acme Corporation is a federal contractor.',
+  description: 'Enter the description for your entity.',
+  required: true,
+  maxLength: 50,
```

Comment:
Max Length is not automatically updating in configurable. However, if I update the value and then type into the field the characters remaining update to the correct value. I'd say thats something that should be updated in the formly type to support that value being changed.

Reply:
Given federalist goes away next week though, maybe toss this into the backlog and deal with it later. What do you think @davereed?